orkbasecxx/CapturePluginProxy.h

```diff
@@ -27,6 +27,9 @@ class DLL_IMPORT_EXPORT_ORKBASE CapturePluginProxy
 	void Run();
 	void Shutdown();
 	void StartCapture(CStdString& party, CStdString& orkuid, CStdString& nativecallid, CStdString& side);
+	void StartStream(CStdString& party, CStdString& orkuid, CStdString& nativecallid);
+	void EndStream(CStdString& party, CStdString& orkuid, CStdString& nativecallid);
+	CStdString GetStream();
```

Comment:
IMO this should be exposed by the Filter and not Proxy. It can implement the `StartCapture`, `StopCapture` to offer the same. Please refactor it to be part of the `LiveStreamFilter` folder